CONTRIBUTING.md

```diff
@@ -21,8 +21,9 @@ To get started quickly, follow the steps below.
 1. Ensure you have Python 3.8+ and [Git](https://git-scm.com/) installed on your system. You can check your Python version by running:
 
    ```bash
-   python --version
+   python --version or python3 --version
    ```
+   If not present, please download latest python version from https://www.python.org/downloads/ and try above command
```

Comment:
If Python 3.8+ is not present, please download the latest version from [python.org](https://www.python.org/downloads/) and try the above command again.

Reply:
Makes sense, let folks how do they wish to download. 
I have made required changes. Please review and merge @Pouyanpi